### ISSUE TITLE
My own trained model damaged.

### ISSUE THREAD

**Issue by wangs0007** [zhanghang1989/PyTorch-Encoding#193]
Hi Dr. Zhang,

First of all I appreciate your excellent work. I am new at using this repo and currently I have some issues while training my own model. When I tried to use the following command (CUDA_VISIBLE_DEVICES=0,1,2,3 python main.py --dataset minc --model deepten_resnet50_minc --batch-size 512 --lr 0.004 --epochs 80 --lr-step 60 --lr-scheduler step --weight-decay 5e-4) to train my own deep-TEN model, a broken model file 'model_best.pth.tar' was generated. I tried to unzip this file but it cannot be done since the file is damaged. The file size does not seem to be right to me either (about 200MB, whereas your pre-trained model.pth is about 100MB). Do you have any idea how may I resolve this? Thank you for your time in advance! 

Shuo
